plugins/git-plugin/src/main/groovy/org/rundeck/plugin/scm/git/BaseGitPlugin.groovy

```diff
@@ -631,7 +631,7 @@ class BaseGitPlugin {
         try {
             push = pushb.call()
         } catch (Exception e) {
-            plugin.logger.debug("Failed push to remote: ${e.message}", e)
+            logger.debug("Failed push to remote: ${e.message}", e)
```

Comment:
this change came from rebase, we can ignore by now, later we can create a new clean PR